packages/dbos-cloud/cli.ts

```diff
@@ -284,11 +284,12 @@ databaseCommands
   .option("-p, --port <number>", "Specify your database port", "5432")
   .option("-W, --password <string>", "Specify password for the dbosadmin user")
   .option("--enable-timetravel", "Enable time travel on the linked database", false)
-  .action(async (dbname: string, options: { hostname: string; port: string; password: string | undefined; enableTimetravel: boolean }) => {
+  .option("--supabase-ref <string>", "Link a Supabase database")
+  .action(async (dbname: string, options: { hostname: string; port: string; password: string | undefined; enableTimetravel: boolean; supabaseRef: string }) => {
```

Comment:
Shouldn't this be `supabaseRef: string | undefined` because the flag is optional?